container/dockershell.sh

```diff
@@ -108,6 +118,18 @@ if [ -z ${ENDPOINT} ]; then
     exit 1
 fi
 
+if [ -z ${DB2_PORT} ]; then
```

Comment:
Just a generic thought, I'm trying to recall the details of the standalone container (maybe @Nintengao can chime in here), but by default I didn't think you set a port nor did we advertise the default ports we selected.

Given that would it improve the user experience to have default ports for `deploymentType standalone` ?  It's a bit of a catch-22 as it implies more maintenance on our part if we do change those ports in the standalone container.

Reply:
Hi Hamdi, there are a few issues that would not make this possible. First, the nodeports are dynamic in CP4D, so it is not possible to have a default port for those. Also, in RHOS 4.X, the remote applications are likely not to have direct access to the worker nodes where Db2 Event store is running. Finally, the user may need to create their own load balancer to get access to the event store ports, and again, those ports will be likely be custom.

Reply:
Hey Christian, but my comment was only for `standalone` i.e. our container only deployment (docker image) not any of the other environments.  Wouldn't your points not be applicable for the standalone container ?

Reply:
Ok, I assume you are talking it to "developer (developer container deployments)" which is not currently external, let's have the discussion when that happens. 

```
                  cp4d (Cloud Pak for Data deployments)
                  wsl (WSL deployments)
                  developer (developer container deployments)
```

Reply:
Sounds good, keep in mind its indirectly documented through this app as it indeed supports `developer` as you cited above.  But we can revisit as you said when we release that officially.